tests/Doctrine/Tests/ORM/Functional/UUIDGeneratorTest.php

```diff
@@ -20,18 +23,28 @@ class UUIDGeneratorTest extends OrmFunctionalTestCase
 {
     use VerifyDeprecations;
 
+    /**
+     * @group dbal2
```

Comment:
Why do we need this groups anyway? Can't the same be achieved with `markTestSkipped`?